cmd/kube-apiserver/app/server.go

```diff
@@ -362,14 +383,13 @@ func (s *APIServer) Run(_ []string) error {
 		if err != nil {
 			glog.Fatalf("experimental API is enabled in runtime config, but not enabled in the environment variable KUBE_API_VERSIONS. Error: %v", err)
 		}
-		if s.ExpStorageVersion == "" {
-			s.ExpStorageVersion = g.Version
+		if _, found := storageVersions[g.Group]; !found {
```

Comment:
likewise-- if this is not set, perhaps that should also trigger the above error?